symbolic-minidump/src/cfi.rs

```diff
@@ -1025,8 +1042,17 @@ impl<'a> CfiCache<'a> {
     /// Load a symcache from a `ByteView`.
     pub fn from_bytes(byteview: ByteView<'a>) -> Result<Self, CfiError> {
         if byteview.len() == 0 || byteview.starts_with(b"STACK") {
-            let inner = CfiCacheInner::V1(CfiCacheV1 { byteview });
+            let inner = CfiCacheInner::Unversioned(CfiCacheV1 { byteview });
             return Ok(CfiCache { inner });
+        } else if byteview.starts_with(&CFICACHE_MAGIC) && byteview.get(7).cloned() == Some(b'\n') {
```

Comment:
It would be better to (1) read the magic, (2) the version, and then (3) the rest based on the version. Comparing with the SymCache format, its header contains further binary data without a newline, for instance.